src/pages/distroSettings/tabs/ProjectTab/types.ts

```diff
@@ -0,0 +1,12 @@
+export interface ProjectFormState {
```

Comment:
I think you'll also want to specify `ProjectFormState` in [`distroSettings/tab/types.ts`](https://github.com/evergreen-ci/spruce/blob/bd3fe088a3a6e926e0d398879ce05adecbf87986/src/pages/distroSettings/tabs/types.ts#L18)!